README.md

```diff
@@ -52,6 +52,16 @@ To change to LimeSurvey configuration, you can mount a Volume into the Container
 
 **Hint**: If this configuration is present before the installation, the LimeSurvey Web Installer will not run automatically.
 
+## Data Encryption
+
+LimeSurvey 4 supports data encryption, this image give you these options:
+
+* Provide a security.php file directly (environment variables)
+* Provide encryption keys for the security.php file (volume)
```

Comment:
Did you accidentally switch the explanation of these two lines?

If I provide the security.php directly, then presumably it's via volume.
If I just provide the keys themselves, the presumably the image configures the security.php file.

Right?

Reply:
I shouldn't do any changes today :facepalm:

Reply:
It's okay :D